experimentals/react-native-uifabric/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "react-native-uifabric",
```

Comment:
Is this win32 specific?  Shouldn't the name be something win32 specific while we are at it?

Reply:
At this time, I'm just copying the existing package over as is. The package is marked as "private", so we can easily rename the package later, until we publish it.

Reply:
I would leave this right now for consistency.  Ideally much of the code here will be shareable between platforms but I think the process should be:

1. get it checked in, mostly as-is
2. get metro working correctly for bundling
3. figure out the right directory structure / package names and migrate into that form

I think we will want to come up with new package names before we make this version the source of truth.